src/tests/electron/tests/automated-checks.view.test.ts

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Application } from 'spectron';
+import { createApplication } from 'tests/electron/common/create-application';
+import { dismissTelemetryOptInDialog } from 'tests/electron/common/dismiss-telemetry-opt-in-dialog';
+import { AutomatedChecksViewSelectors } from 'tests/electron/common/element-identifiers/automated-checks-view-selectors';
+import { CommonSelectors } from 'tests/electron/common/element-identifiers/common-selectors';
+import { testResourceServerConfig } from 'tests/electron/setup/test-resource-server-config';
+import { DEFAULT_ELECTRON_TEST_TIMEOUT_MS } from 'tests/electron/setup/timeouts';
+import * as WebDriverIO from 'webdriverio';
+
+// spectron wraps calls to electron APIs as promises. Unfortunately, only electron typings are used,
```

Comment:
How many of these cases are there? I think it would be nicer for us to add typings files overriding the default electron typings rather than disabling the warning; practically speaking, the "no-floating-promises" warnings are also going to be disabled by this issue, which have historically been a really common source of test flakiness

Reply:
That's an issue I think. 
On one side, my understanding is that we are "forced" to use certaing version of `webdriverio` based on the version of `spectron` we use. I'm not sure why this is or if I'm right about it.
On another side, I tried to add a dev dependency to `@types/webdriverio` but types seems incorrect to me (no Promises for example). I guess we could investigate this a little bit more but that should be a different piece of work, a different PR